modules/kueue-monitoring-pending-workloads-on-demand.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/monitoring-pending-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="monitoring-pending-workloads-on-demand_{context}"]
+= Monitoring pending workloads on demand
+
+[NOTE]
+====
+`VisibilityOnDemand` is a Beta feature enabled by default.
+
+You can disable the feature by setting the `VisibilityOnDemand` feature gate. Check the Installation guide for details on feature gate configuration.
```

Comment:
you can't disable this feature when using the kueue-operator. This section can be removed.